acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -81,6 +81,40 @@ def get_pcp_broker_status(host)
   end
 end
 
+# Start an eventmachine reactor in a thread
+# @return Thread object where the EM reactor should be running
+def start_eventmachine_thread
```

Comment:
Another method name like get_eventmachine_thread would inform readers that this method returns a thread. Then again it wouldn't inform them that that starts a thread...